examples/src/main/scala/org/apache/spark/examples/sql/SQLDataSourceExample.scala

```diff
@@ -56,6 +56,13 @@ object SQLDataSourceExample {
       .option("header", "true")
       .load("examples/src/main/resources/people.csv")
     // $example off:manual_load_options_csv$
+    // $example on:manual_save_options_orc$
+    usersDF.write.format("orc")
+      .option("orc.bloom.filter.columns", "favorite_color")
+      .option("orc.dictionary.key.threshold", "1.0")
+      .option("orc.column.encoding.direct", "name")
+      .save("users_with_options.orc")
```

Comment:
Also, cc @dbtsai .
This doc is only for Spark 3.0.0 since `orc.column.encoding.direct` is added to `master` branch.